crates/bevy_sprite/src/collide_aabb.rs

```diff
@@ -28,7 +29,7 @@ pub fn collide(a_pos: Vec3, a_size: Vec2, b_pos: Vec3, b_size: Vec2) -> Option<C
         } else if a_min.x > b_min.x && a_min.x < b_max.x && a_max.x > b_max.x {
             (Some(Collision::Right), a_min.x - b_max.x)
         } else {
-            (None, 0.0)
+            (Some(Collision::Inside), -f32::INFINITY)
```

Comment:
Since all of these branch are now a `Collision` enum variant, I think we can remove the `Some` around all of them.